activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -49,7 +49,8 @@ def self.initialize_i18n(app)
         when :load_path
           I18n.load_path += value
         when :raise_on_missing_translations
-          setup_raise_on_missing_translations_config(app)
+          strict = value == :strict
+          setup_raise_on_missing_translations_config(app, strict)
```

Comment:
Any reason not to do this?
```suggestion
          setup_raise_on_missing_translations_config(app, value == :strict)
```

Reply:
Thank you. It's just my personal preference. I think the former is slightly less readable and the latter is basically equivalent to the current code. So I want to keep it as it is.